test/test_multispecies_problem.py

```diff
@@ -0,0 +1,139 @@
+import numpy as np
+import festim as F
+from petsc4py import PETSc
+from dolfinx.fem import Constant
+from ufl import exp
+import os
+
+
+def test_multispecies_permeation_problem():
+    """Test running a problem with 2 mobile species permeating through a 1D
+    0.3mm domain, with different diffusion coefficients, asserting that the
+    resulting concentration fields are less than 1% different from their
+    respecitive analytical solutions"""
```

Comment:
No it seems like you are not checking the resulting concentration fields but the resulting fluxes...

```suggestion
    """Test running a problem with 2 mobile species permeating through a 1D
    domain, with different diffusion coefficients, checks that the computed
    permeation flux math the analytical solution"""
```